src/tests/ftest/io/dfuse_bash_cmd.yaml

```diff
@@ -0,0 +1,35 @@
+server_manager:
+    srv_timeout: 500
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-C
+timeout: 900
+server_manager:
+  recreate: False
+server_config:
+    name: daos_server
+    servers:
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+pool:
+  mode: 146
+  name: daos_server
+  scm_size: 1000000000
+  svcn: 1
+  control_method: dmg
+  pool_count: 5
+container:
+  type: POSIX
+  control_method: daos
+  cont_count: 5
+dfuse:
+    mount_dir: /tmp/
```

Comment:
This seems a dangerous value to use, although it should be overwritted by the code as I understand it.